prime-router/build.gradle.kts

```diff
@@ -30,7 +30,7 @@ import java.time.format.DateTimeFormatter
 import java.util.Properties
 
 plugins {
-    kotlin("jvm") version "1.8.22"
+    kotlin("jvm") version "1.9.10"
```

Comment:
Maybe line 43 should also stay in [sync](https://plugins.gradle.org/plugin/org.jetbrains.kotlin.plugin.serialization):
`id("org.jetbrains.kotlin.plugin.serialization") version "1.8.22"`

we also have this defined at line 109
`val kotlinVersion = "1.9.10"`
just to add confusions.

Reply:
good catch, that's logical, kotlin jvm plugin 1.9.10 should go with serializer 1.9.10, let me change it and see...